core/models/user_external.py

```diff
@@ -0,0 +1,16 @@
+from datetime import datetime
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class UserExternalID(models.Model):
```

Comment:
This is used to associate the Django user to the identifier of another application. This should be a one to many 'user' to external application id. I added the display name since we don't link the users immediately or enforce that it must be linked. The username is a more friendly in the ui without needing to query somewhere else. I will make the change if you insist but it was not intended to be a "user".